node/test/graphql/query/config.spec.ts

```diff
@@ -172,6 +172,7 @@ describe('config', () => {
       urlClassifier: '/classifier/v2',
       urlGraphql: '/graphql/v2',
       urlVideo: '/video/v2',
+      urlVideoIdleTips: 'https://youtu.be/xSu1BhuFt8A',
```

Comment:
this url is the same as the default url, so it's not really testing anything. Change to like 'https://some/custom/url'